README.md

```diff
@@ -24,10 +24,12 @@ Unzipped the full parquet data will be approximately 70GB. Each (compressed) chu
 
 ### New Contributor Tips
 
-- A valid and much appreciated contribution is to write any of your learnings, be it by reading research papers or through your interaction with the community on gitter and submitting a Pull Request to the repository. You can submit the PR to the README on the main page or the analysis folder README.
+- Your core focus should be an analysis / contribution riffing off one of the research-question issues. Having said that any open-source project, even the non-traditional ones, need constant improvements from lots of fresh eyes. A valid and much appreciated contribution is to write any of your learnings, be it by reading research papers or through your interaction with the community on gitter and submitting a Pull Request (PR) to the repository. You can submit the PR to the README on the main page or the analysis folder README.
```

Comment:
"Your core focus should be an analysis / contribution riffing off one of the research-question issues."  - this is very outreachy specific. but this repo is not outreachy specific.  Suggest cleaning up my language of "A valid and much appreciated contribution" to make it more generic.

Reply:
Thanks, I have submitted the changes.